src/Tribe/Main.php

```diff
@@ -3807,17 +3807,25 @@ public function isEvent( $event ) {
 		 * @return bool Is it a venue?
 		 */
 		public function isVenue( $postId = null ) {
+			static $is_venue = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: b82bcf4e0ce84f12e779538fb90f2661</sub>

Reply:
Lies.